### PR TITLE
Some refactorings

### PR DESCRIPTION
**Pull request by smoench**
Mostly code cleanups, removing unnecessary code, preventing duplications and applying some symfony best practices for more readable and extendable code.

### CODE REVIEW

**Comment by tolry** [comment]
could you please give at least a short summary of what you're refactoring and why?

**Comment by smoench** [comment]
@tolry a short summary has been written

**Comment by tolry** [comment]
that summary could be taken for almost any refactoring, regardless of what was actually done :smile: I will have a look at the code later

**Comment by smoench** [comment]
Good enough for this moment.

@tolry @larsborn @DavidBadura Please review

File: src/SimpleThings/AppBundle/Controller/CommitController.php [hunk 1]
```diff
@@ -6,21 +6,23 @@
...
 /**
- * @Route("/commit")
+ * @Framework\Route("/commit")
  */
 class CommitController extends Controller
 {
     /**
-     * @Route("/{id}/show", name="commit_show")
+     * @Framework\Route("/{id}/show", name="commit_show")
+     * @Framework\Template()
```
Comment (tolry) [inline src/SimpleThings/AppBundle/Controller/CommitController.php]: Best Practice "Don't use the @Template() annotation to configure the template used by the controller." :smile: I think this is a performance issue.

File: src/SimpleThings/AppBundle/Controller/ProjectController.php [hunk 2]
```diff
@@ -5,102 +5,98 @@
...
      */
     public function lastCommitAction(Project $project)
     {
         /** @var CommitRepository */
-        $repository = $this->get('doctrine')->getRepository('SimpleThings\AppBundle\Entity\Commit');
+        $repository = $this->get('simpspector.app.repository.commit');
         $commit     = $repository->findLastInMaster($project);
 
-        if (!$commit) {
+        if (! $commit) {
```
Comment (tolry) [inline src/SimpleThings/AppBundle/Controller/ProjectController.php]: I am afraid we need to decide on this someday. Let's talk tomorrow?

```php
# PSR-2
if (!$commit) {
     true;
}
# your suggestion
if (!$commit) {
     true;
}
# what Lars & I are using currently
if ( ! $commit) {
     true;
}
```

**Comment by tolry** [comment]
Except for the `@Template()` thingy I like most of the changes. I think the service.xml split is a bit overengineered at this point, but allright with me.

**Comment by larsborn** [comment]
Isn't 
> if (! $commit) {
also conform? Then I would prefer this one.

File: src/SimpleThings/AppBundle/Controller/CommitController.php (hunk 1 above)
Comment (DavidBadura) [inline src/SimpleThings/AppBundle/Controller/CommitController.php]: i like the `@Template()` annotation, we don't have performance issue in this area. So :+1:

File: src/SimpleThings/AppBundle/Controller/ProjectController.php (hunk 2 above)
Comment (DavidBadura) [inline src/SimpleThings/AppBundle/Controller/ProjectController.php]: last one isn't PSR-2 conform

**Comment by tolry** [comment]
Ok, so we leave `@Template()` since it's shorter and we don't have performance issues and keep the used negation syntax. *merge

**Pull request merged**